packages/fast-form-generator-react/src/form/form-item.select.tsx

```diff
@@ -1,6 +1,14 @@
 import * as React from "react";
 import IFormItemCommon from "./form-item";
+import { default as SelectStyles } from "./form-item.select.style";
```

Comment:
you can actually just import styles, there's no need for namespacing because we aren't importing multiple styles into this file